extensions/ql-vscode/src/extension.ts

```diff
@@ -493,6 +494,29 @@ async function activateWithInstalledDistribution(
     }
   }
 
+  async function previewQueryHelp(
+    selectedQuery: Uri
+  ): Promise<void> {
+    const path = selectedQuery ? selectedQuery.path : window.activeTextEditor!.document.uri.fsPath;
+    if(path) {
+      // Create temporary directory
+      const tmpDir = tmp.dirSync();
+      // Get library name and query file name in the format 'libname/queryname.md
+      const mdFilePath = path.split('/').slice(-2).join('/').replace('.qhelp', '.md');
+      const absolutePathToMd = `${tmpDir.name}${mdFilePath}`;
```

Comment:
I would avoid doing path manipulation manually or assuming that the directory separator is always `/`. Let the Node `path` library do this work for you.
```suggestion
      const relativePathToMd = path.basename(path, '.qhelp') + '.md'
      const absolutePathToMd = path.join(tmpDir.name, relativePathToMd)`;
```